.whitesource

```diff
@@ -0,0 +1,8 @@
+{
+  "generalSettings": {
+    "shouldScanRepo": true
+  },
+  "checkRunSettings": {
+    "vulnerableCheckRunConclusionLevel": "failure"
+  }
+}
```

Comment:
Why is this here?

Reply:
it is a configuration file for Whitesource. , "whitesource" is tool which check for vulnerability in  repositories

Reply:
Why did you mark this as resolved? I didn't sign off on this. Please remove this from this PR.

I went to their [web site](https://www.whitesourcesoftware.com), and I got immediately spammed with blinking messages. I found no obvious high level description of how it works. I'm immediately skeptical of this tool.

It is poor decorum on your part to try sneaking stuff like this into unrelated PRs. Please don't do it again on any of my repositories.